frontend/scss/molecules/_m-featured-pages-grid.scss

```diff
@@ -53,7 +73,19 @@
         }
     }
     .featured-pages-grid_row:first-child {
-        border-top: 2px solid #F3F3F3;
+        border-top: 1px solid #e6e6e6;
+
+        @include breakpoint('medium') {
+            border-top: none;
+        }
+
+        @include breakpoint('small') {
+            border-top: none;
+        }
+        @include breakpoint('xsmall') {
+            border-top: none;
+            
+        }
```

Comment:
This can be refactored to use the `+` and `-` notation for breakpoints. If you want the CSS applied to `medium` and anything smaller, do this:

```
@include breakpoint('medium-') {
    border-top: none;
}
```

Then you don't need to duplicate the call for `small` and `xsmall`.